utils/extensions/documentdb.ts

```diff
@@ -1,25 +1,27 @@
+import { Container } from "@azure/cosmos";
 import * as DocumentDb from "documentdb";
 import { Either, left, right } from "fp-ts/lib/Either";
 import { Option } from "fp-ts/lib/Option";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import { mapAsyncIterable } from "io-functions-commons/dist/src/utils/async";
+import {
+  CosmosErrors,
+  toCosmosErrorResponse
+} from "io-functions-commons/dist/src/utils/cosmosdb_model";
 import * as DocumentDbUtilsBase from "io-functions-commons/dist/src/utils/documentdb";
+import * as t from "io-ts";
 
 export * from "io-functions-commons/dist/src/utils/documentdb";
 
 export const deleteDocument = (
```

Comment:
it looks like this method is unused now?

Reply:
It is used here now => [https://github.com/pagopa/io-functions-admin/blob/09edfbcc73bf222d307cefa261202f467a3cc69a/utils/extensions/models/profile.ts#L21](https://github.com/pagopa/io-functions-admin/blob/09edfbcc73bf222d307cefa261202f467a3cc69a/utils/extensions/models/profile.ts#L21)

Reply:
why not` this.container.item(..).delete()` ilke in the other models?